stable/wordpress/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: wordpress
-version: 5.0.4
+version: 5.0.5
```

Comment:
Did you try `helm upgrade`? I don't know if the changes in the ingress are going to break upgradability.

Reply:
yes! It works now